src/main/java/com/autotune/operator/KruizeDeploymentInfo.java

```diff
@@ -38,8 +38,9 @@
  */
 public class KruizeDeploymentInfo {
     private static final Logger LOGGER = LoggerFactory.getLogger(KruizeDeploymentInfo.class);
-    public static String database_username;
-    public static String database_password;
+    public static String database_authentication_type;
```

Comment:
why this change required ? 
this change will affect ROS

Reply:
Stashed for now. Will be updating later on for common auth feature